src/main/java/org/gridsuite/securityanalysis/server/dto/PreContingencyLimitViolationResultDTO.java

```diff
@@ -0,0 +1,30 @@
+package org.gridsuite.securityanalysis.server.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.gridsuite.securityanalysis.server.entities.PreContingencyLimitViolationEntity;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class PreContingencyLimitViolationResultDTO {
+
+    private String subjectId;
+    private String status;
```

Comment:
can you use SecurityAnalysisStatus for status type ?

Reply:
It's not the status of the calculation, but the status of the result that corresponds to this: CONVERGED, MAX_ITERATION_REACHED, SOLVER_FAILED, FAILED; The status of the calculation corresponds to this: NOT_DONE, RUNNING, CONVERGED, DIVERGED, which is sent by the /status controller